pdf-converter/test.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const conveter = require('./pdf-to-image');
+
+
+let event = {
+    "bucket": "docex",

```

Comment:
If this is a test, can you point to a public bucket and documents instead of this private one?

Reply:
Alternatively, this could point out to a bucket defined in an environment variable and copy the documents from the github repo into that bucket before running the tests

Reply:
We have removed this.